.github/workflows/update-components.yaml

```diff
@@ -51,10 +51,13 @@ jobs:
       - name: Create pull request
         uses: peter-evans/create-pull-request@v6
         with:
-          git-token: ${{ secrets.DEPLOY_KEY_TO_UPDATE_STRICT_BRANCH }}
```

Comment:
it took some digging, but i believe i confirmed this wasn't doing anything because the branch on the runner was checked out using an SSH key -- the token isn't necessary here to push to the branch.  Instead it relies on the ssh key already being part of the local git config.